$NetBSD: patch-af,v 1.2 2007/08/12 03:04:33 wiz Exp $

Only define JS_USE_FDLIBM_MATH if the user (generally) or pkgsrc (in
this case) hasn't decided.

--- jslibmath.h.orig	2005-04-29 09:33:47.000000000 -0400
+++ jslibmath.h
@@ -54,6 +54,7 @@
  * by default since there can be problems with endian-ness and such.
  */
 
+#if !defined(JS_USE_FDLIBM_MATH)
 #if defined(_WIN32) && !defined(__MWERKS__)
 #define JS_USE_FDLIBM_MATH 1
 
@@ -82,6 +83,7 @@
 #else
 #define JS_USE_FDLIBM_MATH 0
 #endif
+#endif
 
 #if !JS_USE_FDLIBM_MATH
 
